score-server/src/main/java/bio/overture/score/server/repository/s3/S3DownloadService.java

```diff
@@ -265,30 +296,30 @@ public String getSentinelObject() {
    * Retrieve meta file object
    */
   private S3FetchedObject getObject(String objectId, String objectMetaKey) {
-    String stateBucketName = bucketNamingService.getStateBucketName(objectId);
+    String objectBucketName = bucketNamingService.getObjectBucketName(objectId);
     try {
-      return fetchObject(stateBucketName, objectMetaKey);
+      return fetchObject(objectBucketName, objectMetaKey);
     } catch (AmazonServiceException e) {
       if (e.getStatusCode() == HttpStatus.NOT_FOUND.value()) {
         if (bucketNamingService.isPartitioned()) {
           // Try again with master bucket
           log.warn(
               "Object with objectId: {} not found in {}, objectKey: {}: {}. Trying master bucket {}",
               objectId,
-              stateBucketName,
+              objectBucketName,
               objectMetaKey,
               e,
               bucketNamingService.getBaseStateBucketName());
           try {
-            stateBucketName = bucketNamingService.getBaseStateBucketName(); // use base bucket name
-            val obj = fetchObject(stateBucketName, objectMetaKey);
+            objectBucketName = bucketNamingService.getBaseStateBucketName(); // use base bucket name
+            val obj = fetchObject(objectBucketName, objectMetaKey);
             obj.setRelocated(true);
             return obj;
           } catch (AmazonServiceException e2) {
             log.error(
                 "Failed to get object with objectId: {} from {}, objectKey: {}: {}",
                 objectId,
-                stateBucketName,
+                objectBucketName,
                 objectMetaKey,
                 e);
             if ((e.getStatusCode() == HttpStatus.NOT_FOUND.value()) || (!e.isRetryable())) {
```

Comment:
This is still supposed to be state bucket. We only wanted to swap to objectBucketName for getting meta data when the state file is unavailable.